examples/dotnet/Pulumi.yaml

```diff
@@ -0,0 +1,6 @@
+name: provider-xyz-native
+runtime: dotnet
+plugins:
+  providers:
+    - name: xyz
+      path: ../../bin # Adjust this path to point to your xyz provider binary
```

Comment:
Won't this path always be correct?

Reply:
Yes. Leftover code comment I guess, I can remove that.

Reply:
resolved in https://github.com/pulumi/pulumi-provider-boilerplate/pull/80/commits/fbe3f61f95d8cc26991983ed42d44c6459780161